guides/common/modules/proc_promoting-a-content-view.adoc

```diff
@@ -51,5 +51,24 @@ Now the repository for the Content View appears in all environments.
 ----
 +
 Now the database content is available in all environments.
-
++
 To register a host to your Content View, see {ManagingHostsDocURL}Registering_Hosts_managing-hosts[Registering Hosts] in _{ManagingHostsDocTitle}_.
++
+. Promote Content Views across all lifecycle environments within an organization using the following Bash script:
```

Comment:
Being an alternative to the repetitive steps above, it would make more sense to acknowledge the contrast here.
```suggestion
. Or, you can promote Content Views across all lifecycle environments within an organization using the following Bash script:
```